client/src/modules/tags/tags.ctrl.js

```diff
@@ -41,7 +34,10 @@ function TagsController($uibModal, Tags, Modal,
     vm.errorState = false;
     Tags.read()
       .then(tags => {
-        vm.gridOptions.data = tags;
+        vm.gridOptions.data = tags.map(t => {
+          t.iconColor = { color : t.color, 'font-size' : '14px' };
```

Comment:
Since you do this about 3+ times in this PR, I suggest you put this in a Service.  Like the `TagService`.  Or `ColorService`. 

```js
Tags.formatTagStyle = tag => {
  tag.style = { color : tag.color, 'font-size': '14px' };
};
```

Reply:
@jniles we have these three variants, and i am truggling a bit : 
```js
$ctrl.getTagColor = t => {
  return t ? { color : t.color } : null;
};

function addColorStyle(tag) {
  tag.style = { color : tag.color };
}

function addIconStyle(item) {
  item.iconStyle = { color : item.value };
  return item;
}
```
Finally, the `font-size` doesn't matter.